airflow/api/common/experimental/mark_tasks.py

```diff
@@ -60,6 +62,7 @@ def _create_dagruns(dag, execution_dates, state, run_type):
     return dag_runs
 
 
+@deprecated(reason="Use airflow.api.common.mark_tasks.set_state instead", version="2.2.3")
```

Comment:
If the behaviour of these functions are the same then we should do something like this instead for the whole file:


```
import warnings

from airflow.api.common.mark_tasks import *  # noqa

warnings.warn(
    "This module is deprecated. Please use `airflow.api.common.mark_tasks` instead.", DeprecationWarning, stacklevel=2
)

```

Reply:
Another problem is, I didn't use execution_date at `airflow.api.common.mark_tasks`. Should I add it?

Reply:
I have updated it but left the test file. Helped to verify the re-written functions.